Makefile

```diff
@@ -4,7 +4,7 @@ lint:
 	docker run \
 		-t \
 		--rm \
-		-v ./:/app \
```

Comment:
Hmm this isn't quite right: the `./` should copy the current to the `/app` context so that golangci-lint can pick it up.

What error message are you getting specifically with this?

Reply:
Strange, if I run it with just `-v ./:/app \` I get the docker error shown below:

```
make lint
docker run \
                -t \
                --rm \
                -v ./:/app \
                -w /app \
                golangci/golangci-lint:v1.53.3 \
                golangci-lint run -v
docker: Error response from daemon: create ./: "./" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
See 'docker run --help'.
make: *** [lint] Error 125
```

Reply:
Give this a try:
```diff
diff --git a/Makefile b/Makefile
index 4cf869a..30262b4 100644
--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,12 @@
+ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
 all: lint test build

 lint:
        docker run \
                -t \
                --rm \
-               -v ./:/app \
+               -v "${ROOT_DIR}/:/app" \
                -w /app \
                golangci/golangci-lint:v1.53.3 \
                golangci-lint run -v
```

I think you're right, the volume flag for docker doesn't expect a relative path, it needs to be absolute.

Reply:
Yes that works as expected now, thank you.